pennylane/ops/qubit/matrix_ops.py

```diff
@@ -86,6 +86,30 @@ def _walsh_hadamard_transform(D: TensorLike, n: Optional[int] = None):
     return qml.math.reshape(qml.math.transpose(D), orig_shape)
 
 
+def _compute_udv(a, b):
+    r"""Given the matrices `a` and `b`, calculates the matrices `u`, `d` and `v`
+    of equation (36) [https://arxiv.org/pdf/quant-ph/0504100]:
```

Comment:
```suggestion
    of equation (36) in [arXiv-quant-ph:0504100](https://arxiv.org/pdf/quant-ph/0504100):
```

I think this is how you abbreviate arxiv quantum phys preprints 🤔 ...